ci/docker/Dockerfile.build.armv8

```diff
@@ -27,6 +27,9 @@ ENV HOSTCC gcc
 ENV TARGET ARMV8
 
 WORKDIR /work/deps
+#RUN sed -i '3d' /etc/apt/sources.list
+RUN sed -i '\#deb http://cdn-fastly.deb.debian.org/debian-security jessie/updates main#d' /etc/apt/sources.list
```

Comment:
Could you add a comment explaining why you did this? Also, please remove unused code

Reply:
done